src/open_producten/producttypen/models/externeverwijzingconfig.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+
+
+class ExterneVerwijzingConfig(SingletonModel):
```

Comment:
Having this as a SingletonModel does mean that you can only connect to one instance of these APIs. I'm not sure if that is an issue, but wanted to mention it at least. I feel like we could use `zgw_consumers.Service` to register these external APIs and we could add a FK to these Services on the newly added models, that way we can set up credentials as well and validate that zaaktypen exist as well at some point.

Reply:
Yes, one instance was the requirement, i did implement the zaaktype verwijzing with a service (the other ones do not seem to have one) but because validation wasn't needed it felt a bit overkill, if it this is needed in the future the urlfield can be replaced by a fk to Service fairly easily.